pytest_fixtures/core/contenthosts.py

```diff
@@ -22,8 +22,8 @@ def host_conf(request):
     _rhelver = f"rhel{params.get('rhel_version', settings.content_host.default_rhel_version)}"
     rhel_compose_id = settings.get(f"content_host.hardware.{_rhelver}.compose")
     if rhel_compose_id:
-        conf['rhel_compose_id'] = rhel_compose_id
-    conf['rhel_version'] = settings.content_host.hardware.get(_rhelver).release
+        conf['deploy_rhel_compose_id'] = rhel_compose_id
+    conf['deploy_rhel_version'] = settings.content_host.hardware.get(_rhelver).release
```

Comment:
Why did you rename these two? I don't see them used anywhere. Is it just for consistency?

Reply:
that is an updated workflow argument (the workflow got updated and arguments renamed)